_emails/us/oregon/portland_defund_ppb.md

```diff
@@ -17,34 +17,48 @@ recipients:
-  vote if you aren't willing to make a public and vocal stand to help defund the PPB
-  and increase their accountability and transparency.\n\nThank you for your time,\n[YOUR
-  NAME]\n[YOUR ADDRESS]\n[YOUR EMAIL]\n[YOUR PHONE NUMBER]"
+body: "Hello,
+
+My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.
+
+I am happy to see the proposed cuts in over $15 million to PPB championed by Commissioner Hardesty, and at this time I need to see more. I see the collective call for $50 million in reductions and believe this is insufficient- I want to see an immediate $100 million reduction to the PPB in the 20-21 FY budget cycle. Now is the time to take decisive action, not next year or the next, when public scrutiny will perhaps be less intense. Set a course for the full disbandment of the Portland Police Bureau NOW, or I fear it will never happen.
+
+I am skeptical of Wheeler's ability to steer us clearly as city mayor with his position at the helm of the PPB as Commissioner. The truth is, we need cuts and a commitment to disband, like the cities we see before us setting the stage for such change at this moment in time.
+
+We need to know that we are being taken seriously and that we mean for the PPB to be dismantled in support of building Black futures and community care. The PPB does not protect us equally and we can do better.
+
+Specifically, in accordance with the demands of Care Not Cops and the design to defund and reallocate, I ask for:
+
+1) Immediate elimination of all vacant positions within the PPB, and to fold this funding back into the general and special appropriations funds.\n\n2) Implementation of an immediate hiring freeze, and a commitment to ending any new hires of Portland Police Bureau sworn officers.
+
+3) Immediately eliminate all funds for the following specialty units (those already included in the proposed item are not listed here): Neighborhood Response, Service Coordination Team, Behavioral Health Unit, Air Support Unit and Rapid Response Team (Riot Control)
+  3.a) Redirection of these funds towards anti-violence programs unaffiliated with law enforcement, reparations for Black Portlanders targeted by gang policing, and other investment in Black people to be determined by impacted communities and grassroots groups such as Don’t Shoot Portland & Freedom to Thrive.
+  3.b) Eliminate these positions entirely by reducing the total number of officers, not just by folding them back into general sworn officer positions.
+
+4) Immediately stop policing protests - officers are bringing the riot, not the demonstrators. Even if they stop using tear gas, glass beads, plastic and rubber bullets, and other 'less lethal weapons' they will continue to behave how they know and shift to other ways to enact violence.
```

Comment:
This is nice, but will have to be amended when pr
```suggestion

```

A great point for right now, but would have to be updated later, so I suggest removing until we have time senstive letters.

Reply:
Great point, thank you!